charts/sonarqube/values.yaml

```diff
@@ -225,3 +225,26 @@ serviceAccount:
   # name:
   ## Annotations for the Service Account
   annotations: {}
+
```

Comment:
In order to take advantage of this, wouldn't we need to disable the new configmap postgres-config? Otherwise you could run into a situation where the username is generated by the chart, but you are trying to override with the extra configs.

Reply:
In the case where the username is created externally, and added to a `ConfigMap` which is loaded via `extraConfig`, the username created by the chart is irrelevant- it's always overridden. The `ConfigMap`s we add to the env, if you see the deployment, are always loaded last.

In this case, the data the chart creates is never used, and is always overridden.